package.json

```diff
@@ -62,8 +62,10 @@
     "@stoplight/json-ref-resolver": "1.x.x",
     "@stoplight/types": "4.1.x",
     "@stoplight/yaml": "2.x.x",
+    "@types/js-yaml": "^3.12.1",
     "ajv": "6.x.x",
     "chalk": "^2.4.2",
+    "js-yaml": "^3.13.1",
```

Comment:
```suggestion
```
I believe we can use `parseWithPointers` function exposed by `@stoplight/yaml` package. It's already installed and used by Spectral.

Reply:
Yeah, I can go with it, thanks!